package.json

```diff
@@ -6,8 +6,8 @@
   "author": "Symphony",
   "main": "js/main.js",
   "scripts": {
-    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron .",
-    "demo": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --url=file://$(pwd)/demo/index.html",
+    "dev": "npm run browserify-preload && cross-env ELECTRON_DEV=true electron . --enable-sandbox",
```

Comment:
it should already be enabled...  see: when creating main window: https://github.com/symphonyoss/SymphonyElectron/blob/master/js/windowMgr.js#L48 and also for and child windows.